src/_P036_FrameOLED.ino

```diff
@@ -798,7 +804,7 @@ int left = 24;
   display->setTextAlignment(TEXT_ALIGN_LEFT);
   display->setFont(ArialMT_Plain_16);
   display->setColor(BLACK);
-  display->fillRect(0, 13+TopLineOffset, 128, 64);
+  display->fillRect(0, 13+TopLineOffset, 128, P36_MaxDisplayHeight);
```

Comment:
Same for this 128 value, must it be P36_MaxDisplayWidth or is it a different value?